project.md

```diff
@@ -5,47 +5,168 @@
 
+A DSL is appropriate because there is currently no specification for being able
+to create taiko sheet music. Thus, it'd be useful for some kind of taiko
+language to exist with the intent that the quirks that taiko has but other
+percussion instruments don't are communicated clearly. Additionally, by creating
+a language around taiko notes that can be written down, it may provide people
+who find learning orally difficult a different, more effective medium to learn a
+song.
+
+You could always write it out physically without the need for a program, but having
+a program that you can run has many benefits, including but not excluding easier
+method of editing old sheet music, running sheet music through some interpreter
+to play out the piece, and more.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I've been in the taiko club at the Claremont Colleges since sophomore year and
+it's honestly one of the highlights of my week. Hitting drums relieves your
+stress like nothing else. 
+
+Our club is pretty small; there's only a handful of people from each grade and
+when the seniors graduate, it's a major loss. One of these losses involve not
+knowing how exactly a song goes because we were taught orally by the
+seniors. There are videos of past performances we can look at, but the sound
+quality isn't the greatest nor can we trust that the performers themselves did
+it right. Additionally, we can't really google any of these because again, these
+songs are passed down from instructor to student orally.
+
+Just the other week, we had a minor incident where no one could agree how
+exactly a song went. We ended up deciding on something, but it's very possible
+that we were not correct and therefore altered how the song originally
+went. Thus, I think it would be very helpful to have some kind of physical aid
+to remember how a song goes instead of relying on faulty memory.
+
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Taiko songs into sheet music.
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+I want to make it somewhat similar to MuseScore so that people don't have to
+learn entirely new controls. You would be presented with a GUI where you have a
+sheet of empty staffs in front of you, and then you could click what notes you
+want to put on it. The notes would be colored/shaped differently depending on
+what hand you should play with and where you should hit the drum.
+
+To make it more convenient, I could have a library of common rhythms (such as
+the horse beat, which is a bit of a swing that sounds like a horse galloping)
+that people could click to drop a collection of notes onto a measure, rather
+than doing it one by one. 
+
+Again, I believe this is the right way because musicians are already familiar
```

Comment:
This is good justification for your approach! If there are potential users of this DSL that might be less familiar with sheet music and MuseScore (within the club perhaps) then definitely consider if the MuseScore template can be improved on for these users